platform/options.api/src/org/netbeans/modules/options/OptionsPanel.java

```diff
@@ -964,7 +965,7 @@ private boolean checkSize(Dimension componentSize) {
     }
 
     private static Color getTabPanelBackground() {
-        if( isMetal || isNimbus ) {
+        if( isMetal || isNimbus | isFlatLaf ) {
```

Comment:
isMetal || isNimbus | isFlatLaf  ->   isMetal || isNimbus || isFlatLaf at all places maybe this trio could be replaced an additional isNative() method.

Reply:
Good idea.

Reply:
Why `|` rather than `||`?

Reply:
And surely `isNative` is the opposite meaning?!  Incidentally, would `LookAndFeel::isNativeLookAndFeel` be a better thing to look for rather than using using LaF specific flags?

Reply:
> Why `|` rather than `||`?

Ups, this is a typo.

Reply:
Have fixed the `|` and renamed `isNative` to `useUIDefaultsColors`